moving_zeroes/moving_zeroes.py

```diff
@@ -4,12 +4,41 @@
 '''
 def moving_zeroes(arr):
```

Comment:
Nice job! This is a really clean solution. 

Since last night's lecture was about optimization/writing better solutions, there is an alternate solution to this that uses `O(1)` space-complexity. See if you can find the solution.

Note that this optimization is one where you change your approach, rather than re-work what you have.